index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="style.css">
+    <title></title>
+  </head>
+  <body>
+
```

Comment:
Be sure to add some kind of section (as well as an article template) to which you can append your projects (and a template from which to model them).